src/general/OptionsMenu.cs

```diff
@@ -682,7 +682,9 @@ private void LoadLanguages(OptionButton optionButton)
 
         foreach (var locale in languages)
         {
-            optionButton.AddItem(locale);
+            var currentCulture = Settings.GetCultureInfo(locale);
+            var native = currentCulture.NativeName;
+            optionButton.AddItem(locale + "( " + native + ")");
```

Comment:
That whitespace is placed wrong:
```suggestion
            optionButton.AddItem(locale + " (" + native + ")");
```